test/datetime/tokenParse.test.js

```diff
@@ -66,6 +68,41 @@ test("DateTime.fromFormat() throws if you specify meridiem with 24-hour time", (
   expect(() => DateTime.fromFormat("930PM", "Hmma")).toThrow(ConflictingSpecificationError);
 });
 
+// #714
+test("DateTime.fromFormat() makes dots optional and handles non breakable spaces", () => {
+  function parseMeridiem(input, isAM) {
+    const d = DateTime.fromFormat(input, "hh:mm a");
+    expect(d.hour).toBe(isAM ? 10 : 22);
+    expect(d.minute).toBe(45);
+    expect(d.second).toBe(0);
+  }
+
+  // Meridiem for this locale is "a. m." or "p. m.", with a non breakable space
```

Comment:
Is this a typical representation of meridiems in the `es-ES` locale? (or is this a test case to handle some larger set of nbsp situations?)